src/server.c

```diff
@@ -502,15 +518,32 @@ dictType kvstoreExpiresDictType = {
     kvstoreDictMetadataSize,
 };
 
-/* Command table. sds string -> command struct pointer. */
-dictType commandTableDictType = {
-    dictSdsCaseHash,            /* hash function */
-    NULL,                       /* key dup */
-    dictSdsKeyCaseCompare,      /* key compare */
-    dictSdsDestructor,          /* key destructor */
-    NULL,                       /* val destructor */
-    NULL,                       /* allow to expand */
-    .no_incremental_rehash = 1, /* no incremental rehash as the command table may be accessed from IO threads. */
+/* Command set, hashed by sds string, stores serverCommand structs. */
+hashsetType commandSetType = {
+    hashsetCommandGetKey,       // return key (command full name)
+    dictSdsCaseHash,            // hash function
+    hashsetStringKeyCaseCompare,// case insensitive char* key compare
+    NULL,                       // item destructor
+    NULL,                       // resize allowed callback
+    NULL,                       // rehash start callback
+    NULL,                       // rehash end callback
+    NULL,                       // metadata size callback
+    .instant_rehashing = 1,
+    .userdata = NULL
+};
```

Comment:
It's better to use the `.name = value` style everywhere in these types. Then we don't need the comments, we can skip all the nulls and they don't need to be in a fixed order.

```suggestion
hashsetType commandSetType = {
    .elementGetKey = hashsetCommandGetKey,
    .hashFunction = dictSdsCaseHash,
    .keyCompare = hashsetStringKeyCaseCompare,
    .instant_rehashing = 1,
};
```